ReleaseNotes.md

```diff
@@ -1,5 +1,54 @@
 # Release Notes
 
+## New in Release 2.5.0
+Post-training Quantization:
+- Features:
+  - Added Post-training Quantization support for OpenVINO IR (OpenVINO backend).
+  - Added `"overflow_fix"` parameter (for `quantize(...)` & `quantize_with_accuracy_control(...)` methods) support & functionality. It improves accuracy for optimized model for affected devices.
+  - Added `"model_type"` parameter (for `quantize(...)` & `quantize_with_accuracy_control(...)` methods) support that allows reach better accuracy for specific model architectures (e.g. transformers).
```

Comment:
`model_type` support was in NNCF 2.4.0 for OpenVINO backend (POT) and Torch backend. What was change now?

Reply:
@kshpv, has support for the `model_type` parameter been added to the ONNX backend in this release?

Reply:
> @kshpv, has support for the `model_type` parameter been added to the ONNX backend in this release?

Yes, as well as Torch support